src/core_plugins/kibana/public/management/styles/main.less

```diff
@@ -32,23 +32,30 @@ kbn-management-app {
 }
 
 kbn-management-landing {
-  .management-panel-product {
+  .management-panel {
     margin-bottom: 10px;
 
-    .panel-body {
-      padding-bottom: 30px;
-      position: relative;
-      border-left: 1px solid #E4E4E3;
-      border-right: 1px solid #E4E4E3;
-      border-bottom: 1px solid #E4E4E3;
-    }
   }
 
-  .management-link {
+  .management-panel__heading-icon {
```

Comment:
I think we can delete this selector and its styles. The selector is a duplicate of the one defined on line 82, and the styles don't seem to have a visible impact in the browser.